src/Falco/Response.fs

```diff
@@ -86,12 +86,31 @@ let private writeString (encoding : Encoding) (httpBodyStr : string) (ctx : Http
         let httpBodyBytes = encoding.GetBytes httpBodyStr
         writeBytes httpBodyBytes ctx
 
+[<Obsolete("Response.redirect is deprecated, please use Request.redirectPermanentlyTo or Request.redirectTemporarilyTo respectively")>]

```

Comment:
Can you remove this? The plan is to add these to the master branch, do a final patch version with all the obsolete tags and new funcs.

Reply:
Of course!